src/Stratis.Bitcoin.Features.Api/ApiSettings.cs

```diff
@@ -98,12 +135,15 @@ public ApiSettings(NodeSettings nodeSettings)
         public static void PrintHelp(Network network)
         {
             var builder = new StringBuilder();
+            var defaults = new ApiSettings(NodeSettings.Default(network), savedNodeDefaults);
```

Comment:
I think this is overcomplicating the code without need just print out Network defaults and we are good.